commandapi-platforms/commandapi-bukkit/commandapi-bukkit-test/commandapi-bukkit-test-tests/src/test/java/dev/jorel/commandapi/test/CommandNamespaceTests.java

```diff
@@ -535,7 +535,7 @@ public void testSameCommandNameConflictWithDefaultNamespaceAndCustomNamespace(bo
 				results.set("b");
 			});
 
-		defaultNamespace.register();
+		defaultNamespace.register("minecraft");
```

Comment:
Maybe change this variable name to `minecraftNamespace` instead of `defaultNamespace`.